SECURITY.md

```diff
@@ -5,9 +5,9 @@
 | DOMjudge Version | Supported          | PHP version supported |
 |------------------| ------------------ |-----------------------|
 | main branch      | :warning:          | 8.1-8.4               |
+| 8.4.x            | :white_check_mark: | 8.1-8.4               |
```

Comment:
```suggestion
| 9.0.x            | :white_check_mark: | 8.1-8.4               |
```

Reply:
Isn't that part of the next release? - I am following https://github.com/DOMjudge/domjudge-scripts/blob/main/new_release_howto.md

Reply:
Thanks, I am an idiot